src/styles/card.scss

```diff
@@ -0,0 +1,6 @@
+.card {
+  display: flex;
+  width: $cardWidth;
```

Comment:
Ensure that the variable $cardWidth is defined in your SCSS files. If it is not defined, it will cause a compilation error.